tests/samplers_tests/test_samplers.py

```diff
@@ -124,10 +125,16 @@ def test_categorical(sampler_class, choices):
 
     study = optuna.study.create_study(sampler=sampler_class())
     in_trial_study = InTrialStudy(study)
-    points = np.array([
-        study.sampler.sample_independent(in_trial_study, _create_new_trial(study), 'x',
-                                         distribution) for _ in range(100)
-    ])
+
+    def sample():
+        # type: () -> Any
```

Comment:
I think the type of the return value is required to be `int` because it will be compared with `0` and `len(distribution.choices) - 1`.

Reply:
You're right. Thanks.

Fixed: https://github.com/pfnet/optuna/pull/441/commits/369b52292b2eb3a1ae29537c43a55fbc5414529e